### ISSUE TITLE
Snowmobiles Block Ninja Stars

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#24374]
(WEB REPORT BY: edwarus REMOTE: 206.221.180.138:7777)
# Revision
c3e49c7f3241a7291d2b38975c35f51da2f99573

# Description
Can't hit people on snowmobiles with ninja stars
# Steps to Reproduce
Be a ninja. Throw stars at people on snowmobiles or other carts.
# What you Expected
People to get hit
# What Actually Happened
nothing

**Comment by Probe1** [comment]
I'm pretty sure it applies to more than just snowmobiles but don't quote me

**Comment by strawberrytan** [comment]
yeah vehicles that block pathing like the janicart and secway can tank projectiles
